.editorconfig

```diff
@@ -1,6 +1,6 @@
 root = true
 
-[*]
```

Comment:
oops, good call!

should probably move charset et all to [*] still though, doing it rn